full-service/src/db/transaction_log.rs

```diff
@@ -1968,4 +1970,63 @@ mod tests {
 
         assert_eq!(tx_log.tx, expected_tx_log.tx);
     }
+
+    #[test]
+    fn test_try_from_vec_output_txo_for_transaction_id() {
+        let mut rng: StdRng = SeedableRng::from_entropy();
+        let root_id = RootIdentity::from_random(&mut rng);
+        let recipient_account_key = AccountKey::from(&root_id);
+
+        let num_txos = 5; // txo vector size
+        let num_loops = 5; // number of times to run test
+        let amount = 77;
+
+        for _ in 1..=num_loops {
+            // run test loop
+
+            let mut output_vec: Vec<OutputTxo> = Vec::new();
+
+            for _ in 0..num_txos {
+                // loop to build vector and test try_from()
+
+                let (tx_out, _) = create_test_txo_for_recipient(
+                    &recipient_account_key,
+                    0, // subaddress_index
+                    Amount::new(amount * MOB, Mob::ID),
+                    &mut rng,
+                );
+
+                let output_txo = OutputTxo {
+                    tx_out: tx_out.clone(),
+                    recipient_public_address: recipient_account_key.subaddress(0),
+                    confirmation_number: TxOutConfirmationNumber::default(),
+                    amount: Amount::new(amount * MOB, Mob::ID),
+                    shared_secret: None,
+                };
+
+                output_vec.push(output_txo);
+            }
+
+            let min_public_key = output_vec
+                .iter()
+                .min_by_key(|txo| txo.tx_out.public_key)
+                .unwrap()
+                .tx_out
+                .public_key;
+            let transaction_log_id = TransactionId::try_from(output_vec).unwrap();
+
+            assert_eq!(min_public_key.to_string(), transaction_log_id.0);
```

Comment:
⛏️ I see that there is a `TransactionLog`, but it doesn't appear to be used in this test so perhaps the `_log` should be dropped
```suggestion
            let transaction_id = TransactionId::try_from(output_vec).unwrap();

            assert_eq!(min_public_key.to_string(), transaction_id.0);
```

Reply:
Makes sense.